### PR TITLE
Add files from microverse trials.

### PR DESCRIPTION
**Pull request by Cisco-the-wayword**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        body {
+          font-size:75%;
+        }
+      }
+      .flex{
+        display:flex;
+        flex-direction: column;
+        flex-wrap: nowrap; 
+        align-content: center;   }
+    </style>
```
Comment (Cisco-the-wayword) [inline index.html:22]: Try as much as possible to avoid using inline css.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    
+    -->
+    <h4>Hello World, we are software developers!</h4>
```
Comment (Cisco-the-wayword) [inline index.html:30]: You should try to use only h1 tags for headers ^_^

File: index.html [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        <button href = "#Share" type="button" class="nav-link">Share</button> <br>
+        <button href="#Comment" type="button" class="nav-link">Comment</button> <br>
+      </nav>
+    </header>
+
+    <section id = "Like">
+      <iframe width="560" height="315"  src = "https://www.youtube.com/watch?v=4lncyvgKWt4&list=RD4lncyvgKWt4&start_radio=1" frameborder="0" allow="accelerometer; autoplay;encrypted-media; gyroscope;picture-inipicture" id="video" allowfullscreen></iframe>
+    </section>
+
+    <section id = "Share">
```
Comment (Cisco-the-wayword) [inline index.html:46]: You didn't meet add content to the share section.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+    </section>
+
+    <section id = "Comment">
```
Comment (Cisco-the-wayword) [inline index.html:50]: You should have added a comment box in this section.

**Comment by Cisco-the-wayword** [comment]
https://github.com/Cisco-the-wayword/Eddie_Cisco/issues/new

Hi @Cisco-the-wayword 😊,

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there!

image

Highlights
Great code output overall✔️
Great collaborative work ✔️
Great use of HTML semantic tags ✔

Required Changes ♻️
- Check the comments under the review.
- Try to use more descriptive commit messages.
- You should add commits file by file to be able to carry everyone along.
- Try using a more descriptive and professional READme file (Checkout the [microverse README template](https://github.com/microverseinc/readme-template))
- You should add a gitignore file.
- Files like "index.txt" and "index.babel" should be ignored.👻
- You should have a separate css file for neater code.

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

**Review by awais-amjed — APPROVED** [review]
# STATUS: APPROVED 🥳

Hi @Cisco-the-wayword, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉👏🥇✨

## Highlights

- [x] Great Job on identifying the issues ✔️
- [x] Good use of inline comments ✔️

gif

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**